tests/PyzTest/Zed/Console/_data/cli_sandbox/config/Shared/config_default.php

```diff
@@ -117,6 +117,8 @@
 $config[ZedRequestConstants::TRANSFER_DEBUG_SESSION_FORWARD_ENABLED] = false;
 $config[ZedRequestConstants::TRANSFER_DEBUG_SESSION_NAME] = 'XDEBUG_SESSION';
 
+$config[KernelConstants::SPRYKER_ROOT] = APPLICATION_ROOT_DIR . '/vendor/spryker/spryker/Bundles';
```

Comment:
In split?

Reply:
It was for years there. It is for tests only. So values do not matter.

Reply:
We just have strange failed tests and trying to find the reason.

Reply:
Yeah, but that one shouldnt be in the split tests, only on nonsplit side.
We can clean that up later.